srcpkgs/litterbox/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'litterbox'
+pkgname=litterbox
+version=1.6
+revision=1
+build_style=gnu-configure
+make_build_target="all"
+hostmakedepends="pkg-config"
+makedepends="sqlite-devel libressl-devel"
+short_desc="IRC logger"
+maintainer="Allen Sobot <chilledfrogs@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://git.causal.agency/litterbox/about/"
+distfiles="https://git.causal.agency/$pkgname/snapshot/$pkgname-$version.tar.gz"
```

Comment:
Please don't use `$pkgname` here.

Reply:
Ok, fair enough, what should I use then?

Reply:
I would have thought that the less repetition, the better, but here you go, changed